src/keys/age.rs

```diff
@@ -31,7 +31,7 @@ const MAC_BASE64_LEN: usize = 43;
 fn derive_wrap_key(password: &[u8], salt: &[u8; 16], work_factor: u8, wrap_key: &mut [u8; 32]) {
     // wrap key = scrypt(N = work factor, r = 8, p = 1, dkLen = 32,
     //     S = "age-encryption.org/v1/scrypt" || salt, P = passphrase)
-    let params = ScryptParams::new(work_factor, 8, 1).unwrap();
+    let params = ScryptParams::new(work_factor, 8, 1, 32).unwrap();
```

Comment:
@semenov-vladyslav Just confirming, 32 is the right value?

Reply:
Right, the derived from password via scrypt wrap key is 32-byte key used to protect file key via ChaCha20-Poly1305.